src/main/java/com/mycompany/app/App.java

```diff
@@ -16,13 +16,13 @@
 
 package com.mycompany.app;
 
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
```

Comment:
This is going to look a bit messy as you will have two `com.esri.arcgisruntime` imports at the top and then two below the javafx imports.

Reply:
@colinanderson import order tidied.  Should look better now.  Also done the same on the gradle pull request